### PR TITLE
Add instructions for upgrading tas-adapter

### PR DESCRIPTION
**Pull request by acosta11**
Add instructions for upgrading to a new patch version of Application Service Adapter.

### CODE REVIEW

**Comment by acosta11** [comment]
@tcdowney we based the doc heavily on the corresponding tap upgrade page linked in the story, which excluded guidance on discovering a new patch version. Does it seem like we should add that content? I imagine the assumption in the tap docs is that some other notification would go out to customers instead of implementing a polling system of the image repository / Tanzunet.

**Comment by tcdowney** [comment]
@acosta11 

> ... which excluded guidance on discovering a new patch version. Does it seem like we should add that content?
No, I don't think we need to. Customers are free to develop automation around this for their own use cases, but that is likely to be specific to their specific environments and usecases (i.e. are they relocating images? what automation tooling do they use? etc.).

I do think we should add just a sentence or two at the beginning mentioning that patch versions will be published to TanzuNet. We could even mention that operators can sign up for email alerts from TanzuNet when the product is updated:
https://network.pivotal.io/docs/faq#alerts

Anyways, nothing major and I definitely agree we don't need to describe a polling system or any other sort of automation.

**Comment by tcdowney** [comment]
Other than that this doc looks good, thanks! 👍

File: upgrading.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
+# Upgrading Application Service Adapter
+
+This document describes how to upgrade Application Service Adapter for VMware Tanzu Application Platform.
+
+You can perform a fresh installation of the Application Service Adapter by following the instructions in [Installing Application Service Adapter](install.md).
+
+When upgrading to a new major or minor version of Application Service Adapter, please see that versions' documentation for version-specific configuration and upgrade instructions.
+
+These instructions cover upgrading to a new patch version of the currently installed major minor version of Application Service Adapter.
```
Comment (tcdowney) [inline upgrading.md:9]: > currently installed major minor version of Application Service Adapter
I get what this is saying, but it took me a second to figure out what "major minor version" meant. I'm wondering if we could just simplify this to say "currently installed version", but I get that that is not exactly the same as what this is saying right now. Anyways, I don't really have a better suggestion other than that -- just read oddly to me initially.

**Comment by acosta11** [comment]
@tcdowney thanks for the feedback, we incorporated those suggestions. Should be good for you and @emalm to review.

File: upgrading.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+This document describes how to upgrade Application Service Adapter for VMware Tanzu Application Platform.
+
+You can perform a fresh installation of the Application Service Adapter by following the instructions in [Installing Application Service Adapter](install.md).
+
+When upgrading to a new major or minor version of Application Service Adapter, please see that versions' documentation for version-specific configuration and upgrade instructions.
+
+These instructions cover upgrading to a new patch version of the currently installed version of Application Service Adapter.
+
+>**Note:** Discover new patch versions of the Application Service Adapter on the TanzuNet product page directly, or sign up to receive email alerts when the product is updated: [https://network.tanzu.vmware.com/docs/faq#alerts]().
```
Comment (tcdowney) [inline upgrading.md:11]:
```suggestion
>**Note:** Discover new patch versions of the Application Service Adapter on the TanzuNet product page directly, or [sign up to receive email alerts](https://network.tanzu.vmware.com/docs/faq#alerts) when the product is updated.
```

**Review by tcdowney — APPROVED** [review]
Thanks @acosta11 I made a quick tweak to inline that link, but otherwise LGTM. 👍

**Pull request merged**